backend/api/Controllers/RobotController.cs

```diff
@@ -193,6 +193,8 @@ public async Task<ActionResult<Robot>> PostRobot([FromBody] Robot robot)
         try
         {
             var report = await _isarService.StartMission(robot, echoMissionId);
+            robot.Status = RobotStatus.Busy;
+            await _robotService.Update(robot);
```

Comment:
We might need to verfify from ISAR that the mission was successful in starting before setting robot to busy.

Reply:
fixed